.github/workflows/release.yml

```diff
@@ -32,11 +32,3 @@ jobs:
           context: .
           tags: faasm/faabric:${{ env.TAG_VERSION }}
           build-args: FAABRIC_VERSION=${{ env.TAG_VERSION }}
-      - name: "Build Faabric CLI container"
-        uses: docker/build-push-action@v2
-        with:
-          push: true
-          file: docker/faabric-cli.dockerfile
-          context: .
-          tags: faasm/faabric-cli:${{ env.TAG_VERSION }}
-          build-args: FAABRIC_VERSION=${{ env.TAG_VERSION }}
```

Comment:
The `faabric-cli` container is now redundant as we do everything in the `faabric` container. In addition, the `faabric` container isn't actually used anywhere as it is, so it makes sense to combine them.